test_container/tests/test/standard-flavor/all/pyexasol.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+
+from exasol_python_test_framework import udf
+
+
+class PyexsolConnectionTest(udf.TestCase):
+    # TODO use dsn and credentials injected into the testcase
+    host = "localhost"
+    port = "8888"
+    user = "sys"
+    pwd = "exasol"
+
+    def setUp(self):
+        self.query('create schema pyexasol', ignore_errors=True)
+    
+    def run_secure_pyexasol_connection(self, python_version):
+        self.query(udf.fixindent('''
+            CREATE OR REPLACE {python} SCALAR SCRIPT pyexasol.connect_secure() returns int AS
+            import pyexasol
+            import ssl
+            import os
+            def run(ctx):
+                os.environ["USER"]="exasolution"
```

Comment:
Do we need this enviornment variable?

Reply:
to be honest, not sure, I copied it from the websocket tests